csrankings-x.csv

```diff
@@ -350,6 +350,7 @@ Xu Liu 0001,North Carolina State University,https://xl10.github.io,qhLT67EAAAAJ
 Xu Liu 0006,Xidian University,https://web.xidian.edu.cn/xuliu/,_09bkMgAAAAJ
 Xu Sun 0001,Peking University,https://xusun.org,NOSCHOLARPAGE
 Xu Tang,Xidian University,https://sites.google.com/site/xutanghomepage/,lmskeBMAAAAJ
+Xu Wang 0006,Shenzhen University, https://csse.szu.edu.cn/pages/user/index?id=556,XNDHhIEAAAAJ

```

Comment:
Remove spurious space after comma.

Reply:
Please fix this ASAP - I am completing my quarterly update and will not revisit this after this week until the next quarter.